internal/checker/checker.go

```diff
@@ -27158,6 +27158,8 @@ func (c *Checker) getContextualType(node *ast.Node, contextFlags ContextFlags) *
 		return c.getContextualType(parent, contextFlags)
 	case ast.KindSatisfiesExpression:
 		return c.getTypeFromTypeNode(parent.AsSatisfiesExpression().Type)
+	case ast.KindExportAssignment:
+		return c.tryGetTypeFromTypeNode(parent)
```

Comment:
No test change?

Reply:
Nope!

Reply:
My theory, which I haven't confirmed: in Strada, this code was added by analogy with

```ts
/** @type {X} */
exports.x = (x,y,z) => { }
```

(which is currently broken in Corsa, it's on my list to fix)

But it does nothing because there's no type annotation on `export=`.